lib/web/ui/usercontext.go

```diff
@@ -83,6 +83,8 @@ type userACL struct {
 	AccessRequests access `json:"accessRequests"`
 	// Billing defines access to billing information.
 	Billing access `json:"billing"`
+	// ConnectionDiagnostic defines access to connection diagnostics.
+	ConnectionDiagnostic access `json:"connectionDiagnostic"`
```

Comment:
Is this change related with the check we are doing?

Reply:
i didn't know how literally to take this question :sweat_smile: so i provided both

yes, it's related in that it's required for discovery, so we can pre-check that users have all the necessary perms before allowing user to start the discovery flow

no, in that it's not required for the `does user has at least 1 resource` check, i just sneaked it in b/c it was tiny add in (maybe i should've have?)

Reply:
Got it :+1: 
Thank you